Source/Hurl.RulesetManager/Windows/EditRuleset.xaml

```diff
@@ -0,0 +1,183 @@
+
+                        <TextBlock VerticalAlignment="Center" Text="Browser Name" />
+                        <ComboBox
+                            x:Name="TargetBrowser"
+                            Grid.Column="1"
+                            Margin="0,0,0,12"
+                            ItemsSource="{Binding Browsers, Mode=TwoWay}"
+                            SelectedIndex="{Binding SelectedBrowser, Mode=TwoWay}" />
+                        <TextBlock
+                            Grid.Row="1"
+                            VerticalAlignment="Center"
+                            Text="Alternate Launch:" />
+                        <ComboBox
+                            x:Name="TargetAltLaunch"
+                            Grid.Row="1"
+                            Grid.Column="1"
+                            ItemsSource="{Binding AltLaunches, Mode=TwoWay}" />
+                    </Grid>
+                </ui:Card>
+
+                <ui:Card Padding="12">
+                    <Grid>
+                        <Grid.ColumnDefinitions />
+                        <Grid.RowDefinitions>
+                            <RowDefinition />
+                            <RowDefinition />
+                            <RowDefinition />
+                        </Grid.RowDefinitions>
+
+                        <TextBlock Padding="8" FontSize="16">Add Rule</TextBlock>
+
+                        <Grid Grid.Row="1">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="160" />
+                                <ColumnDefinition Width="1*" />
+                            </Grid.ColumnDefinitions>
+
+                            <ComboBox
+                                x:Name="RuleModeInput"
+                                Margin="8"
+                                SelectedIndex="2">
+                                <ComboBoxItem>Regex</ComboBoxItem>
+                                <ComboBoxItem>Glob</ComboBoxItem>
+                                <ComboBoxItem>Domain</ComboBoxItem>
+                                <ComboBoxItem>String</ComboBoxItem>
+                            </ComboBox>
+
+                            <ui:TextBox
+                                x:Name="RuleInputx"
+                                Grid.Column="1"
+                                Margin="8"
+                                PlaceholderText="*.google.com" />
+                        </Grid>
+
+                        <Grid Grid.Row="2">
+                            <ui:Button
+                                Width="80"
+                                Margin="8"
+                                Padding="12,8"
+                                HorizontalAlignment="Left"
+                                Appearance="Transparent"
+                                Icon="{ui:SymbolIcon Info16}">
+                                Info
+                            </ui:Button>
+                            <ui:Button
```

Comment:
Are there in the same Grid cell?

Reply:
They are aligned differently, one to left and one to right